packages/flutter_test/lib/src/binding.dart

```diff
@@ -467,6 +467,11 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     HitTestResult hitTestResult, {
     TestBindingEventSource source = TestBindingEventSource.device,
```

Comment:
Would `{TestBindingEventSource source,}` instead of `{TestBindingEventSource source = TestBindingEventSource.device,}` have the same effect here and reduce the confusion?

Reply:
I think so. @Hixie

Reply:
I'm guessing for an `enum`, leaving it `null` is not preferred.